spec.html

```diff
@@ -15076,10 +15076,10 @@ <h1>[[GetPrototypeOf]] ( ): either a normal completion containing either an Obje
         <dd>a Proxy exotic object _O_</dd>
       </dl>
       <emu-alg>
+        1. Perform ? ValidateNonRevokedProxy(_O_).
+        1. Let _target_ be _O_.[[ProxyTarget]].
         1. Let _handler_ be _O_.[[ProxyHandler]].
-        1. If _handler_ is *null*, throw a *TypeError* exception.
         1. Assert: _handler_ is an Object.
```

Comment:
These asserts seem a little redundant now.

Reply:
They're certainly not necessary, but still arguably a helpful primer for the subsequent GetObject. I have no strong feelings about them.